src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/AddPersonnelFormDisciplinesDropDown.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { SearchableDropdown, TextInput, SearchableDropdownOption } from '@equinor/fusion-components';
+import Personnel from '../../../../../../../models/Personnel';
+import useBasePositions from '../../../../../../../hooks/useBasePositions';
+
+export type PersonnelFormDisciplinesDropDown = {
+    onChange: (changedPerson: Personnel) => void;
+    selectedField: string;
+    item: Personnel;
+    disabled: boolean;
+};
+
+const AddPersonnelFormDisciplinesDropDown: React.FC<PersonnelFormDisciplinesDropDown> = ({
+    onChange,
+    selectedField,
+    item,
+    disabled
+}) => {
+
+    const { basePositions, isFetchingBasePositions, basePositionsError, } = useBasePositions();
+
+    const options = React.useMemo(() => {
+        if (isFetchingBasePositions || basePositionsError)
+            return []
+
+        return basePositions.map(s => ({
```

Comment:
We should extract the discipline from each base position and filter out duplicates and use as options instead of the base position itself